File: DDYDLS_CineClubLocalModel/Models/Movie.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DDYDLS_CineClubLocalModel.Models
{
    public class Movie
    {
        public int Id_Movie { get; set; }
        public string Name { get; set; }
        public string Synopsis { get; set; }
        public int Year { get; set; }
        public Ratings Rating { get; set; }
        public int AvgRating { get; set; }
        public int RatingForUser { get; set; }
        public int TMDB_ID { get; set; }
    }
}

File: DDYDLS_CineClubFrontEnd/src/app/models/movie.ts
import { Rating } from "./rating";

export class Movie {
    id_Movie : number;
    name : string;
    id_Studio : number;
    synopsis : string;
    year : number;
    rating : Rating;
    avgRating : number;
    ratingForUser : number;
    UrlIMG ?: string ;
    TMDB_Id ?: number ;
    EN_name ?: string
    Rottenrating ?: number;
    avgratingEXT ?: number;
    TMDBrating ?: number;
}
